source/product-overview/mattermost-v11-changelog.md

```
@@ -13,6 +13,162 @@
+### Upgrade Impact
+
+#### Database Schema Changes
+ - The following schema changes are included in the v11.9 release. No database downtime is expected for this upgrade. See the [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html) for more details.
+   - Added
+
+#### config.json
+New setting options were added to ``config.json``. Below is a list of the additions and their default values on install. The settings can be modified in ``config.json``, or the System Console when available.
+  - **Changes to Enterprise Advanced plan:**
+    - Under ``AccessControlSettings`` in ``config.json``, added ``EnableChannelPolicyIndicators`` configuration setting to control whether channel access attribute indicators are shown in the channel members list and invite dialog. Default is ``true``.
+  - Added a new Enterprise configuration setting, TeamSettings.LockProfileFieldsForEmailUsers (System Console > Site Configuration > Users and Teams), which prevents users who sign in with email and password from changing their first name, last name, and username ("name_and_username"), or additionally their nickname, position, and profile picture ("all"). System Admins are exempt, and empty first/last names can be filled in once. When enabled, users with the Invite Users permission can pre-set the first name, last name, and username on email invitations; the POST /api/v4/teams/{team_id}/invite/email endpoint accepts a new optional "profiles" field, and the System Console user detail page now supports editing a user's first and last name.
+ - Added ``TeamSettings.LockProfileFieldsForEmailUsers`` configuration setting.
```

markdownlint-cli2 (0.23.0)

[warning] Inconsistent indentation for list items at the same level
Expected: 2; Actual: 1

(MD005, list-indent)